vivisect/qt/memory.py

```diff
@@ -407,6 +407,24 @@ def getExprTitle(self):
 
         return title
 
+    def _getRenderVaSize(self):
```

Comment:
Who calls this one? Or this just a touch up for future code?

Reply:
envi/qt/memory.py VQMemoryWindow._renderMemory() calls this.  since this is a subclass of VQMemoryWindow, overriding this seemed the cleanest place to modify this to look for a Location

Reply:
Ahhh. I see how it's set up now. Excellent.